exist-versioning-release.md

```diff
@@ -150,48 +231,72 @@ You will require a system with:
             <!-- eXist-db Github Release -->
             <server>
                 <id>github</id>
-                <privateKey>[Github Personal access tokens]</privateKey>
+                <privateKey>YOUR-Github-ACCESS-TOKEN</privateKey>
             </server>
         </servers>
+        
+        <profiles>
+           <profile>
+               <id>existdb-release-signing</id>
+               <properties>                   
+                      
+                   <existdb.release.key>YOUR-GPG-KEY-ID</existdb.release.key>
+                   <existdb.release.public-keyfile>/your/part/to/.gnupg/pubring.gpg</existdb.release.public-keyfile>
+                   <existdb.release.private-keyfile>/your/part/to/.gnupg/secring.gpg</existdb.release.private-keyfile>
+                   <existdb.release.key.passphrase>YOUR-GPG-PASSWORD</existdb.release.key.passphrase>
+
+                   <existdb.release.keystore>/your/path/to/.keystore</existdb.release.keystore>
+                   <existdb.release.keystore.pass>YOUR-KEYSTORE-PASSWORD</existdb.release.keystore.pass>
+                   <existdb.release.keystore.key.alias>YOUR-KEYSTORE-ALIAS</existdb.release.keystore.key.alias>
+                   <existdb.release.keystore.key.pass>YOUR-KEYSTORE-PASSWORD</existdb.release.keystore.key.pass>
+               </properties>
+           </profile>
+        </profiles>
+        <activeProfiles>
+            <activeProfile>existdb-release-signing</activeProfile>
+        </activeProfiles>
     </settings>
+```
+
+1. Merge any outstanding PRs that have been reviewed and accepted for the milestone eXist-5.3.0.
+
+2. Create a new clone of the branch you want to release from into a new folder.
+   In the example below the `develop` branch is cloned into the folder `release`
+
+
+    ```bash
+    git clone \ 
+      -b develop \
+      git@github.com:eXist-db/exist.git \
```

Comment:
Should likely use the HTTPS URLs here as that is what the release process is geared up for, i.e. see the SCM section in the `pom.xml`. If you insist on sticking with SSH here then you need an earlier section stating that you need a public SSH key registered with GitHub account and exist-db org.

Reply:
Also HTTPS is more likely to work for anyone working from within a constrained/fire-walled environment

Reply:
The interesting thing is 2FA here. Also as I really never use HTTPS, I really can't tell how that will end up working. Wouldn't you then need to tell people to create an access token and use that here?

Reply:
We have always used HTTPS even with 2FA in the past. It's very simple, use your same GitHub username, but instead of your "password" you just paste in a PAC in its place. If you have 2FA enabled (which eXist-db do) and you are working with HTTPS, which is the default (and I know other eXist-db developers also use it) - then you will almost certainly always be working with a PAC that you previously generated (and maybe stored in your Apple KeyChain so you don't have to remember it ;-)). But sure, a bit about needing a PAC up-front might make sense - I don't know if people need that much hand-holding then, I think we can assume that developers have knowledge of how to use GitHub.